### PR TITLE
Onion address XSOAR indicator type fix

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11740

## Description
Onion address indicator regex fix.

### CODE REVIEW

**Review by ShahafBenYakir — APPROVED** [review]
Approving the ExtractDomainAndFQDNFromUrlAndEmail.py part

File: Packs/Base/TestPlaybooks/playbook-Onion_address_extraction_test.yml [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+      "yes":
+      - "9"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotExists
+          left:
+            value:
+              simple: ExtractedIndicators
```
Comment (ssokolovich) [inline Packs/Base/TestPlaybooks/playbook-Onion_address_extraction_test.yml:424]: ExtractedIndicators.Onion_Address
```suggestion
              simple: ExtractedIndicators
```

Should be more accurate and dedicated for that specific type you're checking

File: Packs/Base/TestPlaybooks/playbook-Onion_address_extraction_test.yml
Comment (ssokolovich) [inline Packs/Base/TestPlaybooks/playbook-Onion_address_extraction_test.yml:1]: As for the else paths for tasks # 4/6/11 use the `PrintErrorEntry` script.

Comment (Ni-Knight) [reply Packs/Base/TestPlaybooks/playbook-Onion_address_extraction_test.yml:1]: Done

File: Packs/Base/TestPlaybooks/playbook-Onion_address_extraction_test.yml (hunk 1 above)
Comment (Ni-Knight) [reply Packs/Base/TestPlaybooks/playbook-Onion_address_extraction_test.yml:424]: Im not checking Onion addresses here, im checking that no other indicatro type was created by mistake.

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**